From: Jeff Mahoney <jeffm@suse.com>
Subject: [PATCH] profile: profile_event_{,un}register shouldn't be exported unconditionally

 profile_event_register and profile_event_unregister are static inlines
 without CONFIG_PROFILING defined. This causes a section type conflict.

Signed-off-by: Jeff Mahoney <jeffm@suse.com>

---

 kernel/profile.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/kernel/profile.c	2007-07-08 19:32:17.000000000 -0400
+++ b/kernel/profile.c	2007-08-27 14:01:19.000000000 -0400
@@ -200,10 +200,10 @@ EXPORT_SYMBOL_GPL(unregister_timer_hook)
 EXPORT_SYMBOL_GPL(task_handoff_register);
 EXPORT_SYMBOL_GPL(task_handoff_unregister);
 
-#endif /* CONFIG_PROFILING */
 
 EXPORT_SYMBOL_GPL(profile_event_register);
 EXPORT_SYMBOL_GPL(profile_event_unregister);
+#endif /* CONFIG_PROFILING */
 
 #ifdef CONFIG_SMP
 /*
